Gemfile

```diff
@@ -1,20 +1,16 @@
-source "https://rubygems.org"
+source 'https://rubygems.org'
 
 # Declare your gem's dependencies in news_tematica.gemspec.
 # Bundler will treat runtime dependencies like base dependencies, and
 # development dependencies will be added by default to the :development group.
 gemspec
 
-gem 'haml-rails'
-gem 'premailer-rails'
+gem 'pg'
 
 # Para archivos estáticos
 gem 'paperclip'
```

Comment:
I think that If paperclip is a dependency should go in the gemspec, and if not, should not be here either.